README.md

```diff
@@ -21,6 +21,23 @@ aznfswatchdog will update the DNAT rules appropriately.
 This package picks a free private IP which is not in use by user's machine and mount the NFSv3 share using that IP and
 create a DNAT rule to route the traffic from the chosen private IP to original endpoint IP.
 
+> **Mount helper program for a secure communication channel for Azure File NFS mounts.**
```

Comment:
Instead of "program", how about "use case".  It is the same "program" for both v3 and v4.  Also, use "Azure Files NFSv4" (and NFSv3) consistently to distinguish the different use cases.